release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+*STDOUT*. The *fluentd* plug-in transforms the log message and sends it to
+Elasticsearch (ES).
+
+Enable `openshift_logging_install_eventrouter` by setting it to `true`. It is
+off by default. *Eventrouter* is deployed to the default namespace. Collected
+information is in operation indices of ES and only cluster administrators have
+visual access.
+
+See the
+link:https://github.com/openshift/origin-aggregated-logging/blob/master/docs/proposals/kube_events_design_doc.md[design
+documentation] for more information.
+
+==== Kibana Dashboards for Operations Are Now Shareable
+
+This allows {product-title} administrators the ability to share saved Kibana
+searches, visualizations, and dashboards.
+
+When `openshift_logging_elasticsearch_kibana_index_mode` is set to `shared_ops`, one
+`admin` user can create queries and visualizations for other `admin` users.
+Other users can not see those same queries and visualizations.
+
+When `openshift_logging_elasticsearch_kibana_index_mode` is set to `unique`,
+users can only see saved queries and visualizations they created. This is the
+default behavior.
+
+See
+xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Aggregating
+Container Logs] for more information.
+
+[[ocp-37-removed-es-copy-method]]
+==== Removed ES_Copy Method for Sending Logs to External ES
+
+`ES_Copy` was replaced with the *secure_formard* plug-in for fluentd to send
+logs from fluentd to external fluentd (that can then ingest into ES). `ES_COPY`
+is removed from the installer and the documentation.
+
+When `openshift_installer` is run for logging to upgrade to 3.7, the installer
+now checks for `ES_COPY` in the inventory and fails the upgrade with:
+
+----
+msg: The ES_COPY feature is no longer supported. Please remove the variable from your inventory
+----
+
+See
+xref:../install_config/aggregate_logging.adoc#fluentd-log-external-elasticsearch[Aggregating
+Container Logs] for more information.
+
+[[ocp-37-expose-es-as-a-route]]
+==== Expose Elasticsearch as a Route
+
+By default, Elasticsearch (ES) deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. This enables a route for external
+access to ES for those tools that want to access its data.
+
+You now have direct access to ES using only your OpenShift token and have the
+ability to provide the external ES and ES Ops hostnames when creating the server
+certificate (similar to Kibana). Ansible tasks now simplify route deployment.
+
+[[ocp-37-removed-metrics-and-logging-deployers]]
+==== Removed Metrics and Logging Deployers
+
+The metrics and logging deployers bare now replaced with `playbook2image` for
+`oc cluster up` so that `openshift-ansible` is used to install logging and
+metrics:
+
+----
+$ oc cluster up --logging --metrics
+----
+
+Check metrics and pod status:
+
+----
+$ oc get pod -n openshift-infra
+$ oc get pod -n logging
+----
+
+[[ocp-37-prometheus]]
+==== Prometheus (Technology Preview)
```

Comment:
- Why is this not connected to [[ocp-37-master-prometheuh-endpoint-coverage]]
    - How does this differ from that section (above). 
- Feels like its lacking information (given its a major point to the release). 
- What is this, how do I use it, why is it important?